WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteSourceManager.kt

```diff
@@ -59,41 +55,28 @@ class MySiteSourceManager @Inject constructor(
 
     fun build(coroutineScope: CoroutineScope, siteLocalId: Int?): List<LiveData<out PartialState>> {
         return if (siteLocalId != null) {
-            allSupportedMySiteSources.map { source ->
-                source.build(coroutineScope, siteLocalId)
-                        .addDistinctUntilChangedIfNeeded(!mySiteDashboardPhase2FeatureConfig.isEnabled())
-            }
+            allSupportedMySiteSources.map { source -> source.build(coroutineScope, siteLocalId) }
         } else {
-            siteIndependentSources
-                    .map { source ->
-                        source.build(coroutineScope)
-                                .addDistinctUntilChangedIfNeeded(!mySiteDashboardPhase2FeatureConfig.isEnabled())
-                    }
+            siteIndependentSources.map { source -> source.build(coroutineScope) }
         }
     }
 
     fun isRefreshing(): Boolean {
-        if (mySiteDashboardPhase2FeatureConfig.isEnabled()) {
-            val source = if (selectedSiteRepository.hasSelectedSite()) {
-                allSupportedMySiteSources
-            } else {
-                siteIndependentSources
-            }
-            source.filterIsInstance(MySiteRefreshSource::class.java).forEach {
-                if (it.isRefreshing() == true) {
-                    return true
-                }
+        val source = if (selectedSiteRepository.hasSelectedSite()) {
+            allSupportedMySiteSources
+        } else {
+            siteIndependentSources
+        }
+        source.filterIsInstance(MySiteRefreshSource::class.java).forEach {
+            if (it.isRefreshing() == true) {
+                return true
             }
         }
         return false
     }
 
     fun refresh() {
-        if (mySiteDashboardPhase2FeatureConfig.isEnabled()) {
-            refreshAllSources()
-        } else {
-            refreshSubsetOfAllSources()
-        }
+        refreshAllSources()
```

Comment:
❓ Since we are calling `refreshAllSources` from the `refresh` function. Does it make sense to replace the existing usages of `refresh` with `refreshAllSources` and remove `refresh` completely?

Reply:
Makes sense! But I did it with a slight variation. I think `refresh()` implicitly means we want to refresh all sources unless we call out explicitly that we want to refresh a subset of sources. So I moved `refreshAllSources()` logic to refresh and removed `refreshAllSources()` completely in 86ad6f6.

Reply:
@ashiagr Nicely done.